packages/path_provider/path_provider_android/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.1.1
+
+- Added implementation of getDownloadsDirectory for Android
```

Comment:
Please see https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style discussion of verbs and punctuation.

Also, please use the same list bullet format as the rest of the file.